.devcontainer/devcontainer.json

```diff
@@ -38,5 +45,6 @@
 		"jebbs.plantuml"
 	],
 	// Run Bash script in .devcontainer directory
-	"postCreateCommand": "/bin/bash ./.devcontainer/post-install.sh"
+	// Use 'onCreateCommand' to run commands on container creation.
```

Comment:
I would add the post-create and post-install scripts even if they don't do anything right now so they will be there in the future.

A best practice is to run apt-update / upgrade in post-install and pull your base docker images in post-start

Not required, just a best practice